dev/src/write-batch.ts

```diff
@@ -509,6 +543,51 @@ export class WriteBatch {
     return this.commit_();
   }
 
+  /**
+   * Commits all pending operations to the database and verifies all
+   * preconditions. Fails the entire write if any precondition is not met.
+   *
+   * The writes in the batch are not applied atomically and can be applied out
+   * of order.
+   *
+   * @private
+   */
+  async batchCommit(commitOptions?: {
+    requestTag?: string;
```

Comment:
Since this is private and one argument, I think we don't need to have an options object here.

Reply:
removed usage of `requestTag`